db/common/src/main/java/org/commonjava/indy/db/common/AbstractStoreDataManager.java

```diff
@@ -272,6 +285,40 @@ protected boolean store( final ArtifactStore store, final ChangeSummary summary,
 
         final StoreKey storeKey = store.getKey();
 
+        logger.warn("Storing {} using operation lock: {}", store, opLocks);
+
+        if(configuration != null) {
+            ArtifactStoreValidateData validateData = null ;
+            try {
+
+                logger.warn("=> [AbstractStoreDataManager] Check Config: " + configuration );
+
+                if(internalFeatureConfig.getSslValidation() ) {
+                    validateData = storeValidator.validate(store);
+                    logger.warn("=> [AbstractStoreDataManager] Validate ArtifactStoreValidateData: " + validateData);
```

Comment:
We'll have to turn this log level down before we release this. It'll generate quite a bit of traffic on the logs.